apps/project-1/prod/terraform/locals.tf

```diff
@@ -4,7 +4,7 @@ locals {
 	project = "project-1"
 	project_purpose = "Web Application"
 	domain = "yourdomain.com.br"
-  droplet_user = "root"
+  	droplet_user = "root"
```

Comment:
I don't understand this change :thinking:

Reply:
It is just "cosmetic". My editor was blaming this line because it was not at the same level as the other configurations.

Reply:
I can revert it if you prefer.

Reply:
What do you think @rafaelbmateus ?

Reply:
I would like to understand why the editor said that, because it's on the same level
I think that y can revert it. We can open another PR with this objective in the future.

Reply:
Ack, I'll revert it. I use the [Terraform LS](https://github.com/hashicorp/terraform-ls) along with [Terraform mode](https://github.com/emacsorphanage/terraform-mode) and [Terraform LSP](https://emacs-lsp.github.io/lsp-mode/page/lsp-terraform-ls/) in Emacs to automatically update the `.tf` structure for me.

Reply:
Reverted this commit.